### PR TITLE
scenario card arrows added

### PR DESCRIPTION
**Pull request by PawanKaur**
issue #147 

There are problems of showing the arrows with the Recovered compartment.

### CODE REVIEW

**Review by NXXR — CHANGES_REQUESTED** [review]
There are a few things that need to be addressed before merging:
- [x] run formatting (`npm run format`) to make sure the code is properly formatted and passes the DevOps pipeline
- [x] fix the lince height of the compartments (the changes result in scenario card compartment entries with more space between rows which lead to a mismatch between the header rows and the compartment rows:  
    grafik
- [ ] address the changes suggested/mentioned by Martin in the last Sprint meeting (changing the "positive" direction/color of the arrows & adding a neutral option, changeing how incremental values are handled)

**Comment by PawanKaur** [comment]
As discussed, to change condition of grey arrow. It should show when the change is not significant or less than 1%

**Comment by PawanKaur** [comment]
- Stagnation rule applied, grey arrow will show up if the change is more than 3%. 
- Arrows also showing up on the basis of compartment value for those cases where RKI values are 0.

File: frontend/src/components/Sidebar/DistrictMap.tsx [hunk 1]
```diff
@@ -14,7 +14,7 @@ import HeatLegendEdit from './HeatLegendEdit';
 import {HeatmapLegend} from '../../types/heatmapLegend';
 import {LockOpen} from '@mui/icons-material';
 import LoadingContainer from '../shared/LoadingContainer';
-
+import HomeIcon from '@mui/icons-material/Home';
```
Comment (NXXR) [inline frontend/src/components/Sidebar/DistrictMap.tsx:17]: These changes have nothing to do with this PR.
```suggestion
```

File: frontend/src/components/Sidebar/DistrictMap.tsx [hunk 2]
```diff
@@ -32,7 +32,7 @@ interface IRegionPolygon {
 
 export default function DistrictMap(): JSX.Element {
   const [geodata, setGeodata] = useState<GeoJSON.GeoJSON | null>(null);
-  //const selectedDistrict = useAppSelector((state) => state.dataSelection.district);
+  const selectedDistrict = useAppSelector((state) => state.dataSelection.district);
```
Comment (NXXR) [inline frontend/src/components/Sidebar/DistrictMap.tsx:35]: These changes have nothing to do with this PR.
```suggestion
  //const selectedDistrict = useAppSelector((state) => state.dataSelection.district);
```

File: frontend/src/components/Sidebar/DistrictMap.tsx [hunk 3]
```diff
@@ -256,7 +256,10 @@ export default function DistrictMap(): JSX.Element {
     isFetching,
     legend,
   ]);
-
+  useEffect(() => {
+    // search polygon list for selected district
+    // apply hover effect or highlight hovering effect.
+  }, [selectedDistrict]);
```
Comment (NXXR) [inline frontend/src/components/Sidebar/DistrictMap.tsx:262]: These changes have nothing to do with this PR.
```suggestion

```

File: frontend/src/components/Sidebar/DistrictMap.tsx [hunk 4]
```diff
@@ -275,6 +278,13 @@ export default function DistrictMap(): JSX.Element {
           />
         </Grid>
         <Grid item container justifyContent='center' xs={1}>
+          <HomeIcon
+            color='primary'
+            fontSize='large'
+            onClick={() => {
+              dispatch(selectDistrict({ags: '00000', name: t('germany'), type: ''}));
+            }}
+          />
```
Comment (NXXR) [inline frontend/src/components/Sidebar/DistrictMap.tsx:287]: These changes have nothing to do with this PR.
```suggestion
```

**Review by NXXR — CHANGES_REQUESTED** [review]

**Review by NXXR — APPROVED** [review]

**Pull request merged**